### PR TITLE
Close me please : Updated fr (French) translation

### PR DESCRIPTION
**Pull request by StelFux**
Commit made via [Stringlate](https://lonamiwebs.github.io/stringlate/)

### CODE REVIEW

**Comment by Strubbl** [comment]
@StelFux can you please rebase your commit on current master?

@tcitworld or @nicosomb can you please review the translation?

**Comment by di72nn** [comment]
Rebased or not, the entries are reordered, so it will probably be a PITA to review. Maybe there is a tool to show a difference for strings.xml, so the changes can be incorporated manually (without all the ordering changes). In that case there is no need for a rebase.

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -206,6 +215,9 @@
     <string name="pref_name_ui_disableTouch_keyCode">Disable touch input button</string>
     <string name="pref_desc_ui_disableTouch_keyCode">Press here to configure the button used to disable/enable touch input</string>
 
+    <string name="pref_name_ui_scrollOverBottom">Faire défiler vers le bas pour marquer comme lu</string>
+    <string name="pref_desc_ui_scrollOverBottom">Appuyez ici pour configurer le nombre de fois que vous avez besoin de \"faire défiler vers le bas\" pour marquer un article comme lu</string
```
Comment (Strubbl) [inline app/src/main/res/values-fr/strings.xml]: you are missing a closing angle bracket at the end of this line

**Comment by Strubbl** [comment]
@StelFux can you please not include the two commits from tcit in this branch? So that we only have your change of the french strings.xml in this pull request.

@tcitworld @nicosomb the changes for the file app/src/main/res/values-fr/strings.xml are clean and reviewable now (7 newly translated strings). Can you please have a look?

**Comment by Strubbl** [comment]
We want to keep the .travis.yml file. I fear this pull request with Stringlate is too screwed up now. Sorry, I have no idea how to clean this up.

May i ask you to use our translation tooling to cleanly add your translations? There is no need to register, but you can if you want:
https://hosted.weblate.org/projects/wallabag/android-app/

**Comment by di72nn** [comment]
I can do a clean merge manually (not in this PR, though), if the translation is ok.

**Pull request closed without merging**